setup.py

```diff
@@ -74,6 +74,57 @@ def _dbg(s, tp=None):
         print(s)
 
 
+def _find_library_dirs_ldconfig():
+    # Based on ctypes.util from Python 2
+
+    if sys.platform.startswith("linux") or sys.platform.startswith("gnu"):
+        if struct.calcsize('l') == 4:
+            machine = os.uname()[4] + '-32'
+        else:
+            machine = os.uname()[4] + '-64'
+        mach_map = {
+            'x86_64-64': 'libc6,x86-64',
+            'ppc64-64': 'libc6,64bit',
+            'sparc64-64': 'libc6,64bit',
+            's390x-64': 'libc6,64bit',
+            'ia64-64': 'libc6,IA-64',
+            }
+        abi_type = mach_map.get(machine, 'libc6')
+
+        # XXX assuming GLIBC's ldconfig (with option -p)
+        # XXX Alpine Linux uses musl that can't print cache
```

Comment:
Why the `XXX`s?

Reply:
But why prefix these comments with `XXX`?

Reply:
Just an awareness note.

Reply:
Let's remove the `XXX`s then, they're just normal comments.

Reply:
For sure. Any more comments so I can do it together.